apps/studio/components/interfaces/Database/Hooks/HTTPRequestFields.tsx

```diff
@@ -123,6 +123,14 @@ const HTTPRequestFields = ({
             type="number"
             actions={<p className="text-foreground-light pr-2">ms</p>}
           />
+          <Input
+            id="max_retries"
+            name="max_retries"
+            label="Number of retries"
+            labelOptional="Between 0 and 5"
+            type="number"
```

Comment:
Better to use `type="range"` and set the min/max? (I know there's a check on submission) https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range

Reply:
I think that would work only if @supabase/frontend has a better component for it that displays also the current value. As I think it should be consistent for both the retries and the timeout picker. 

Seemed messy/less clear when I tried it:
<img width="723" alt="image" src="https://github.com/supabase/supabase/assets/5036432/239a77ae-a1d7-4ad3-9033-c415023ebb5e">


Code used in the screenshot:
````html
          <Input
            id="timeout_ms"
            name="timeout_ms"
            label="Timeout"
            labelOptional="Between 1000ms to 5000ms"
            type="range"
            min="1000"
            max="5000"
            step="100"
            actions={<p className="text-foreground-light pr-2"></p>}
          />
          <Input
            id="max_retries"
            name="max_retries"
            label="Number of retries"
            labelOptional="Between 0 to 5"
            min="0"
            max="5"
            step="1"
            type="range"
            actions={<p className="text-foreground-light pr-2"></p>}
          />
````